templates/hotgrid.hbs

```diff
@@ -6,8 +6,8 @@
             <div class="hotgrid-grid-inner clearfix">
                 
                 {{#each _items}}
-                <div class="hotgrid-grid-item">
-                    <button class="base hotgrid-item-image hotgrid-item-states-{{#if _graphic.hasImageStates}}image{{else}}css{{/if}} {{#if _isVisited}}visited{{/if}} {{#if ../_hasRoundedImages}}border-radius{{/if}}" role="button" aria-label="{{#if _graphic.title}}{{_graphic.title}}.{{/if}}{{#if _graphic.alt}} {{_graphic.alt}}.{{/if}}{{#if _isVisited}} {{../../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
+                <div class="hotgrid-grid-item" data-index={{@index}}>
+                    <button class="base hotgrid-item-image hotgrid-item-states-{{#if _graphic.hasImageStates}}image{{else}}css{{/if}} {{#if _isVisited}}visited{{/if}} {{#if ../_hasRoundedImages}}border-radius{{/if}}" role="button" aria-label="{{#if _graphic.title}}{{_graphic.title}}.{{/if}}{{#if _graphic.alt}} {{_graphic.alt}}.{{/if}}{{#if _isVisited}} {{../../_globals._accessibility._ariaLabels.visited}}.{{/if}}" data-index={{@index}}>
```

Comment:
path to `visited` aria-label should be `../_globals._accessibility._ariaLabels.visited`

could you add `aria-haspopup="dialog"` and remove `role="button"`?